### ISSUE TITLE
Snipaste like Painter - Area Selection

### ISSUE THREAD

**Issue by Zackhardtoname** [flameshot-org/flameshot#1498]
**Is your feature request related to a problem?**
No

The painter tool is only able to highlight a small horizontal bar. For my display, it is always only 1/4 of the line-height... I think it'd be nice if we can click on one area and stop in another area like the box tool with fill the background with some color.

**Comment by mmahmoudian** [comment]
@Zackhardtoname there is no tool called "painter". Do you mean "marker" with which you can highlight text? If so, you can change the line height with either scrolling or alternatively press space and change the size through the "Active thickness" slider.

**Comment by Zackhardtoname** [comment]
Yes. I think the ability to select an area would be very convenient since it is effectively changing the size in real-time.
I really appreciate Snipaste and your wonderful work!

**Comment by mmahmoudian** [comment]
@Zackhardtoname I don't understand the connection between your last comment and your initial post. Would you please clarify.

**Comment by borgmanJeremy** [comment]
Would adding an opacity slider fulfil your need? Then you could use the box tool and adjust the opacity as talked about in #1582

**Comment by Zackhardtoname** [comment]
Yes! I think that'd work great!
